<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderWindow</class>
 <widget class="QWidget" name="OrderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pencarian order</string>
  </property>
  <widget class="EventTable" name="tvOrderList">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>800</width>
     <height>161</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QTableView" name="tvOrderDetail">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>800</width>
     <height>201</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnChoose">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>440</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Pilih</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>440</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Batal</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrint">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>47</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Invoice</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtInvoiceFilter">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="Masukkan invoice untuk mencari transaksi"/>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="placeholderText">
    <string>Invoice</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>47</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Tanggal</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dtOrderFilter">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>d/M/yyyy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSearch">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cari</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="pbWait">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>801</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>0</number>
   </property>
   <property name="value">
    <number>-1</number>
   </property>
   <property name="format">
    <string>Sedang mengambil data...</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EventTable</class>
   <extends>QTableView</extends>
   <header>eventtable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
